src/Monocle/Backend/Test.hs

```diff
@@ -41,11 +41,11 @@ fakeDate = [utctime|2021-05-31 10:00:00|]
 fakeDateAlt = [utctime|2021-06-01 20:00:00|]
 
 alice, bob, eve, fakeAuthor, fakeAuthorAlt :: Author
-alice = Author "alice" "a"
-bob = Author "bob" "b"
-eve = Author "eve" "e"
-fakeAuthor = Author "John" "John"
-fakeAuthorAlt = Author "John Doe/12" "review.opendev.org/John Doe/12"
+alice = Author "alice" "a" mempty
```

Comment:
to avoid these changes, we could define a pattern synonym like this:

```haskell
pattern Author :: Text -> AuthorWithGroup
pattern Author name = AuthorWithGroup name mempty
```
… and then rename the data constructor to `AuthorWithGroup`